### ISSUE TITLE
Is this project still maintained? 

### ISSUE THREAD

**Issue by marcelfolaron** [socioboard/Socioboard-5.0#419]
I'd love to use it but it seems there hasn't been any activity since 2022? 
Thanks.

**Comment by wokawoka** [comment]
I would also be very interested in knowing that

**Comment by TechnologyClassroom** [comment]
@marcelfolaron @wokawoka I have been following the project for some time to see if it becomes stable to the point where I could deploy it in production. Development seems to have slowed down. From other reports, it seems that version 5 does not work. Version 4 might still work, but it is unmaintained.

@vaughngx4 put in a lot of work to get socioboard 5 to run and eventually made a [new project](https://github.com/vaughngx4/OpenSMM) with a similar goal, but I am not sure if it works yet. If you look through the issues related to `linux` and `docker`, you will inevitably find the state of the project and all of the ways to get it running if you want to try Socioboard yourself.

I also found [Ferdium](https://github.com/ferdium/ferdium-app) which seems to have integrations working for just about all social media platforms, but I have not had the chance to try it yet.

**Comment by vaughngx4** [comment]
Hi all, OpenSMM works but only for Twitter and only the `development` branch currently due to build errors in the main branch and me not having the time to clean up the dev branch to merge to main. I am working on getting a CI/CD pipeline running for the project using Jenkins. 

I haven't managed to get any of the actual posting working in SocioBoard 5 but the easiest way to try it would be the Docker setup. Just be sure to read the various ReadMe files because it can get slightly confusing for people new to networking/hosting. Be sure to leave `AMember` disabled as this is a paid third party web application that was originally required to used v5. I have (probably?) disabled most, if not all of it when I was working on the project so long as the .env file is set correctly.

**Comment by TechnologyClassroom** [comment]
@vaughngx4 Before Twitter rebranded to X, they disabled posting through the API. Does OpenSMM use a different method?

**Comment by vaughngx4** [comment]
@TechnologyClassroom Ah if that's true then OpenSMM won't work anymore, I haven't tested it in a while. Will have to look into storing Twitter logins encrypted so that we can move to browser based posting. I have experience with puppeteer so we will most likely move all WIP API usage to browser based using Chromium, although we might run into issues with media. I will post something on our Discord server soon.

**Comment by DE127** [comment]
Some social networks' APIs have changed and some features no longer work. I have found a few paid sources that can work similarly to stackposts,...

**Comment by TechnologyClassroom** [comment]
I think [Ferdium](https://github.com/ferdium/ferdium-app) is the only free option that is maintained.

**Comment by saturnphp** [comment]
> Some social networks' APIs have changed and some features no longer work. I have found a few paid sources that can work similarly to stackposts,...

can u give some examples

**Comment by Xsmael** [comment]
> I think [Ferdium](https://github.com/ferdium/ferdium-app) is the only free option that is maintained.

This is really misleading ferdium is a completely different tool, and doesn't seem to have integrated social media platforms APIs to be able to post or schedule post right from the app.

**Comment by josephrooks** [comment]
> @Xsmael Look at https://github.com/ferdium/ferdium-recipes/tree/main/recipes

Ferdium is great but it is not an alternative to this feature-wise. Using multiple platforms manually from within a desktop app is not the same thing as having a content management tool for scheduling and posting to those platforms.

**Comment by TechnologyClassroom** [comment]
I haven't tried it, but it looked like it scratched the itch from the outside.

**Comment by josephrooks** [comment]
> I haven't tried it, but it looked like it scratched the itch from the outside.

Yeah, given how many platforms are hostile toward third party apps and API usage at this point, even though Ferdium is not in the same class of apps functionally, it still may be the best option for many people who are being forced to go back to managing all of this crap manually in the respective sites again.

**Comment by TechnologyClassroom** [comment]
Do you know if these missing features have been turned into feature requests on the Ferdium issue tracker? (Since I haven't used Ferdium, I don't know if this is a dumb question.)

**Comment by josephrooks** [comment]
> Do you know if these missing features have been turned into feature requests on the Ferdium issue tracker? (Since I haven't used Ferdium, I don't know if this is a dumb question.)

I doubt that will happen, because the scope of Ferdium is pretty much limited to running "app" websites within Ferdium to have them all in one place, not adding features to any of those sites.

**Comment by jumbophp** [comment]
me personally I've ended up implementing myself

**Comment by Xsmael** [comment]
> me personally I've ended up implementing myself

Would you mind sharing how you went about it ?

**Comment by sureshbabu1995** [comment]
Hello Team,

We are continuing to maintain and plan for upcoming changes. Soon, we will launch updates on this platform, and we may introduce a new version. Based on our previous discussions, issues, and other significant points, we will strive to make the platform accessible to both technical and non-technical users.

**Comment by luminous11111** [comment]
How how the progress gone so far, is it plug and play now? Thank you @sureshbabu1995
